go.mod

```diff
@@ -3,6 +3,7 @@ module sigs.k8s.io/controller-runtime
 go 1.15
 
 require (
+	github.com/davecgh/go-spew v1.1.1
```

Comment:
Why is this now required, I can not see it referenced anywhere?

Reply:
leftover